modules/ROOT/partials/misc/setting-the-license.adoc

```diff
@@ -18,12 +18,30 @@ tinymce.init({
 
 If the developer intends to self-host {productname} under a commercial license, a valid license key must be provided. Customers who have purchased a self-hosted-eligible license for {productname} will find their license key in the link:https://www.tiny.cloud/auth/login/[account portal]. To purchase a commercial license, see available options on the link:{pricingpage}/[pricing page].
 
+The {productname} 8 commercial license key will have a `+T8LK:+` prefix.
+
 ==== Example
 
 [source,js]
 ----
 tinymce.init({
   selector: 'textarea',  // change this value according to your HTML
-  license_key: '<your-license-key>'
+  license_key: 'T8LK:...'
+});
+----
+
+=== Use {productname} under GPL with Premium Features
+
+If the developer intends to self-host the {productname} editor under the GPL license and agree to its terms, whilst also self-hosting premium features, a valid license key must be provided. Customers who have purchased a self-hosted-eligible license for {productname} will find their license key in the link:https://www.tiny.cloud/auth/login/[account portal]. To purchase a commercial license, see available options on the link:{pricingpage}/[pricing page].
+
+This type of license key is specifically for open source projects that need premium features while maintaining GPL compliance.
+
+The {productname} 8 commercial license key will have a `+GPL+T8LK:+` prefix.
```

Comment:
Is it intentional that this is included here but in no other examples (such as [license-key-property.adoc](https://github.com/tinymce/tinymce-docs/pull/3753/files#diff-ae53a59e5e803a7fcb2c7d69329c682b4e6c56dc3a80ea44e8a5297dd06b430d))?